app/views/journeys/capitalallowances/balancingAllowance/BalancingAllowanceView.scala.html

```diff
@@ -36,8 +35,7 @@
         @govukErrorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "value_0")))
     }
 
-    @heading(messages("balancingAllowance.title"), None, extraClasses = "no-padding-top")
-
+    @heading("balancingAllowance.title")
```

Comment:
I've gone through each of these pages after changing to make sure that it hasn't ruined the formatting but it still matches the prototype fine.